spk/tt-rss/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tt-rss
-SPK_VERS = 20160102
-SPK_REV = 10
+SPK_VERS = 20170422
```

Comment:
My opinion is to remove this "timestamp" package version concept and instead use upstream project tag - here 17.4 - even if it "temporarily" prevents smooth upgrade...